frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -161,6 +161,17 @@ action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.
 action.GitMachete.BaseSlideOutBranchAction.notification.title.slide-out-success.without-delete.HTML=<html>Branch <b>{0}</b> has been slid out but not deleted</html>
 
 
+action.GitMachete.BaseSquashAction.dialog.title=Squashed Commit Message
+action.GitMachete.BaseSquashAction.dialog.label=Edit message for squashed commit
+action.GitMachete.BaseSquashAction.text=S_quash\u2026
+action.GitMachete.BaseSquashAction.branch-is-root=Squash disabled because branch ''{0}'' is a root
+action.GitMachete.BaseSquashAction.notification.title=Commits Squashed
+action.GitMachete.BaseSquashAction.not-enough-commits=Squash disabled because branch ''{0}'' has {1} commits. Two or more commits are required.
```

Comment:
Whoops this can say `1 commits` in particular... pls provide `{2}` for the optional `s` ending